foundry.toml

```
@@ -1,6 +1,21 @@
 [profile.default]
+solc_version = "0.8.19"
```

```suggestion
solc_version = "0.8.23"
```
perhaps a more recent-ish solidity release?